docker/local-universe/Dockerfile.static

```diff
@@ -0,0 +1,30 @@
+## The versions in this file are hardcoded for audit purposes, and should not be changed.
```

Comment:
Alternately, depending on the org, some orgs may want an older version and some may want a new version, and we would want to be able to present both to them (via different versions of the image, I think).